contracts/event/LemonadeEventAttestation.sol

```diff
@@ -0,0 +1,179 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.19;
+
+import "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import "@ethereum-attestation-service/eas-contracts/contracts/IEAS.sol";
+import "@ethereum-attestation-service/eas-contracts/contracts/resolver/SchemaResolver.sol";
+import "@ethereum-attestation-service/eas-contracts/contracts/resolver/examples/AttesterResolver.sol";
+
+import "./ILemonadeEventAttestation.sol";
+import "./resolvers/EventHostSchemaResolver.sol";
+import "./resolvers/TicketIssuingSchemaResolver.sol";
+
+contract Event {}
+
+contract LemonadeEventAttestation is
+    OwnableUpgradeable,
+    ILemonadeEventAttestation
+{
+    using ECDSA for bytes;
+    using ECDSA for bytes32;
+
+    bytes32 public eventCreatorSchemaId;
+    bytes32 public eventDetailSchemaId;
+    bytes32 public eventCohostSchemaId;
+    bytes32 public ticketTypeSchemaId;
+    bytes32 public ticketSchemaId;
+
+    IEAS internal eas;
```

Comment:
This should be prefixed with an underscore, see https://docs.soliditylang.org/en/latest/style-guide.html#underscore-prefix-for-non-external-functions-and-variables